src/pages/superadmin/tableComponent/index.tsx

```diff
@@ -335,6 +375,92 @@ export const MyTable = ({
+                    <EuiText
+                      className="statusText"
+                      style={{
+                        color: isProviderPopoverOpen ? color.grayish.G10 : ''
+                      }}
+                    >
+                      Provider:
+                    </EuiText>
+                    <div className="filterStatusIconContainer">
+                      <MaterialIcon
+                        className="materialStatusIcon"
+                        icon={`${
+                          isProviderPopoverOpen ? 'keyboard_arrow_up' : 'keyboard_arrow_down'
+                        }`}
+                        style={{
+                          color: isProviderPopoverOpen ? color.grayish.G10 : ''
+                        }}
+                      />
+                    </div>
+                  </StatusContainer>
+                }
+                panelStyle={{
+                  border: 'none',
+                  boxShadow: `0px 1px 20px ${color.black90}`,
+                  background: `${color.pureWhite}`,
+                  borderRadius: '0px 0px 6px 6px',
+                  marginTop: '0px',
+                  marginLeft: '20px'
+                }}
+                isOpen={isProviderPopoverOpen}
+                closePopover={closeProviderPopover}
+                panelPaddingSize="none"
+                anchorPosition="downLeft"
+              >
+                <ProviderContainer>
+                  <ProvidersListContainer>
+                    {providers && providers.length > 0 ? (
+                      providers.map((provider: Bounty) => (
+                        <ProviderContianer key={provider.owner_id}>
+                          <ProviderInfo>
+                            <ProviderImg
+                              src={provider.owner_img || `/static/person_placeholder.png`}
+                              alt="provider"
+                            />
+                            <Providername>
+                              {provider.owner_alias || provider.owner_pubkey}
+                            </Providername>
+                          </ProviderInfo>
+                          <Checkbox
+                            type="checkbox"
+                            name={provider.owner_alias}
+                            checked={
+                              providersCheckboxSelected &&
+                              providersCheckboxSelected.some(
+                                (p: Bounty) => p.owner_id === provider.owner_id
+                              )
+                            }
+                            onChange={() =>
+                              handleProviderSelection && handleProviderSelection(provider)
+                            }
+                          />
+                        </ProviderContianer>
+                      ))
+                    ) : (
+                      <p>No provider with such alias</p>
```

Comment:
No provider available, should be message when the provider length is empty

Reply:
@elraphty Addressed